php/WP_Mock/Filter.php

```diff
@@ -22,7 +22,7 @@ class Filter extends Hook
     public function apply($args)
     {
         if (isset(static::$filtersWithAnyArgs[ $this->name ])) {
-            $args = array_values(static::$filtersWithAnyArgs);
+            $args = [ static::$filtersWithAnyArgs[ $this->name ] ];
```

Comment:
Instead of doing this, what if we updated this line https://github.com/10up/wp_mock/blob/411d1c3cde08a995142ac735e2b83ad8fb308829/php/WP_Mock/Filter.php#L74

To be like this:

```diff
- static::$filtersWithAnyArgs[ $this->name ] = $random_value;
+ static::$filtersWithAnyArgs[ $this->name ] = [ $random_value ];
```

Then in this line we could reference it a bit more intuitively:

```suggestion
            $args = static::$filtersWithAnyArgs[ $this->name ];
```

Reply:
@agibson-godaddy This makes perfect sense. I agree.